kratos/python/add_processes_to_python.cpp

```diff
@@ -138,6 +139,14 @@ void  AddProcessesToPython(pybind11::module& m)
     .def("ClearNeighbours",&FindGlobalNodalNeighboursProcess::ClearNeighbours)
     .def("GetNeighbourIds",&FindGlobalNodalNeighboursProcess::GetNeighbourIds)
     ;
+
+    py::class_<FindGlobalNodalElementalNeighboursProcess, FindGlobalNodalElementalNeighboursProcess::Pointer, Process>
+        (m,"FindGlobalNodalElementalNeighboursProcess")
+            .def(py::init<const DataCommunicator&, ModelPart&>())
+    .def("ClearNeighbours",&FindGlobalNodalElementalNeighboursProcess::ClearNeighbours)
+    .def("GetNeighbourIds",&FindGlobalNodalElementalNeighboursProcess::GetNeighbourIds)
```

Comment:
Do we need to expose those methods?

Reply:
yes, GetNeighbourIds is needed in the tests, and ClearNeighbours is needed if one wants to cleanup the memory

Reply:
Ok
Meanwhile, I see ClearNeighbours as an stand alone process

Reply:
I don't agree about the standalone process. we would need a different one for each of the neighbours type